data/outfits/ship-stats.txt

```diff
@@ -17,6 +17,12 @@ outfit "Ruin-The-Fun 100000 Cargo Space"
 	"cargo space" 100000
 	description "Having that much cargo space will attract Pirate raids."
 
+outfit "Ruin-The-Fun 100000 Bunk Space"
+	"display name" "100000 Bunk Space"
+	category "Special"
+	thumbnail "outfit/bunk room"
+	bunks 1000000
+	description "It's like a sardine can."
```

Comment:
Good one that was missing.

However while for outfit and cargo the attribute names are `cargo space` and `outfit space`, the one for bunks is just `bunks` (no "space"). So I would be calling it "Ruin-The-Fun 100000 Bunks"

Reply:
I can make an exception for those and but them in `RTF God Outfits`, because they are the most basic stats and one of them, "Outfit Space" also adds engine / weapon space.